packages/account/src/Configs/personal-details-config.ts

```diff
@@ -210,9 +209,7 @@ export const personal_details_config = ({
         },
     };
 
-    // Need to check if client is high risk (only have SVG i.e. China & Russia)
-    // No need to get additinal details when client is high risk
-    if (!is_high_risk_client_for_mt5 && real_account_signup_target !== 'maltainvest') {
+    if (real_account_signup_target !== 'maltainvest') {
```

Comment:
Instead of adding these fields dynamically here, cant we update the personal-details config to support  these fields in svg landing company as well?
<img width="756" alt="Screenshot 2024-03-20 at 10 16 47 AM" src="https://github.com/binary-com/deriv-app/assets/84661147/2a23bb78-d114-42eb-bee4-3fb962681338">

Reply:
I personally think not to touch the config yet this requirement did change in the past recently. We have a single place where we can modify and I think it's better

Reply:
Yes i agree with @shahzaib-deriv. They might change it again